temperature/crud.py

```diff
@@ -0,0 +1,65 @@
+from sqlalchemy.orm import Session
+
+from temperature import models, schemas
+
+
+def create_temperature(
+    db: Session, temperature: schemas.TemperatureCreateUpdate
+) -> models.Temperature:
+    db_temperature = models.Temperature(**temperature.model_dump())
```

Comment:
Consider adding error handling for database operations, such as catching exceptions during commit, to prevent application crashes and provide meaningful error messages.